R/like.R

```diff
@@ -1,12 +1,12 @@
 # Intended for use with a data.table 'where'
 # Don't use * or % like SQL's like.  Uses regexpr syntax - more powerful.
 # returns 'logical' so can be combined with other where clauses.
-like = function(vector, pattern, ignore.case = FALSE, fixed = FALSE) {
+like = function(vector, pattern, ignore.case = FALSE, fixed = FALSE, perl = FALSE) {
```

Comment:
I don't think it's common enough to need another infix for it, but we might as well add `useBytes` as an argument too? Comes up occasionally when working with messy strings.

Or maybe even just change the signature to `function(vector, pattern, ...)` and pass it on, although `grepl` is more limited than `grep`...